src/codeu/chat/server/Server.java

```diff
@@ -42,76 +42,76 @@
-				Serializers.INTEGER.write(out, NetworkCode.SERVER_INFO_RESPONSE);
-				Uuid.SERIALIZER.write(out, info.version);
-			}
-		});
+    this.commands.put(NetworkCode.SERVER_INFO_REQUEST, new Command() {
+      public void onMessage(InputStream in, OutputStream out) throws IOException {
+        Serializers.INTEGER.write(out, NetworkCode.SERVER_INFO_RESPONSE);
+        Uuid.SERIALIZER.write(out, info.version);
+      }
+    });
 
-		// New Message - A client wants to add a new message to the back end.
-		this.commands.put(NetworkCode.NEW_MESSAGE_REQUEST, new Command() {
-			@Override
-			public void onMessage(InputStream in, OutputStream out) throws IOException {
+    // New Message - A client wants to add a new message to the back end.
+    this.commands.put(NetworkCode.NEW_MESSAGE_REQUEST, new Command() {
+      @Override
+      public void onMessage(InputStream in, OutputStream out) throws IOException {
 
-				final Uuid author = Uuid.SERIALIZER.read(in);
-				final Uuid conversation = Uuid.SERIALIZER.read(in);
-				final String content = Serializers.STRING.read(in);
+        final Uuid author = Uuid.SERIALIZER.read(in);
+        final Uuid conversation = Uuid.SERIALIZER.read(in);
+        final String content = Serializers.STRING.read(in);
 
-				final Message message = controller.newMessage(author, conversation, content);
+        final Message message = controller.newMessage(author, conversation, content);
 
-				Serializers.INTEGER.write(out, NetworkCode.NEW_MESSAGE_RESPONSE);
-				Serializers.nullable(Message.SERIALIZER).write(out, message);
+        Serializers.INTEGER.write(out, NetworkCode.NEW_MESSAGE_RESPONSE);
+        Serializers.nullable(Message.SERIALIZER).write(out, message);
 
-				timeline.scheduleNow(createSendToRelayEvent(author, conversation, message.id));
-			}
-		});
+        timeline.scheduleNow(createSendToRelayEvent(author, conversation, message.id));
+      }
+    });
 
-		// New User - A client wants to add a new user to the back end.
-		this.commands.put(NetworkCode.NEW_USER_REQUEST, new Command() {
-			@Override
-			public void onMessage(InputStream in, OutputStream out) throws IOException {
+    // New User - A client wants to add a new user to the back end.
+    this.commands.put(NetworkCode.NEW_USER_REQUEST, new Command() {
+      @Override
+      public void onMessage(InputStream in, OutputStream out) throws IOException {
 
-				final String name = Serializers.STRING.read(in);
-				final User user = controller.newUser(name);
+        final String name = Serializers.STRING.read(in);
+        final User user = controller.newUser(name);
 
-				Serializers.INTEGER.write(out, NetworkCode.NEW_USER_RESPONSE);
-				Serializers.nullable(User.SERIALIZER).write(out, user);
-			}
-		});
+        Serializers.INTEGER.write(out, NetworkCode.NEW_USER_RESPONSE);
+        Serializers.nullable(User.SERIALIZER).write(out, user);
+      }
+    });
 
 		// New Conversation - A client wants to add a new conversation to the back
```

Comment:
The lines below also need fixing. 

I like to take a quick look at the diff as a sanity check, to catch things like these. (Using either `git diff` or the "files changed" tab of the GitHub pull request)